scripts/README.md

```diff
@@ -1,15 +1,84 @@
-# Using scripts
+## Steps to use scripts
```

Comment:
Could we start at the top with brief paragraph telling me what this page is about (before this heading).

Reply:
Okay, will do that.